content/en/docs/tasks/observability/distributed-tracing/zipkin/index.md

```diff
@@ -41,12 +41,11 @@ $ istioctl dashboard zipkin
 
     {{< boilerplate trace-generation >}}
 
-1.  From the top panel, select a service of interest (or 'all') from the **Service Name** drop-down list and click
-    **Find Traces**:
+1.  From the search panel, click on the plus sign. Select `serviceName` from the first dropdown and `productpage.default` from second dropdown and click the search icon:
```

Comment:
```suggestion
1.  From the search panel, click on the plus sign. Select `serviceName` from the first drop-down list, `productpage.default` from second drop-down, and then click the search icon:
```

Reply:
I find this instruction confusing because doesn't seem to have any correlation with the following diagram. The text should in some way match what's shown in the following diagram.